docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  ms: &ms
+    build:
+      context: .
```

Comment:
To control image name might be nice to be explicit about this for the ms build inside ```docker-compose.yml``` maybe ```image: msfconsole``` or ```image: metasploitframework```, thoughts?